cli/command/formatter/disk_usage.go

```diff
@@ -150,6 +158,43 @@ func (ctx *DiskUsageContext) Write() {
 		}
 	}
 	ctx.postFormat(tmpl, &volumeContext{v: types.Volume{}})
+
+	// And download cache
+	ctx.Output.Write([]byte("\nDownload Cache space usage:\n\n"))
+	tmpl, err = ctx.startSubsection(defaultDiskUsageTableFormat)
+	err = ctx.contextFormat(tmpl, &diskUsageDownloadCacheContext{
+		DownloadCacheUsage: ctx.DownloadCacheUsage,
+	})
+	if err != nil {
+		return
+	}
+	ctx.postFormat(tmpl, &diskUsageDownloadCacheContext{})
+}
+
+type diskUsageDownloadCacheContext struct {
+	HeaderContext
+	types.DownloadCacheUsage
+}
+
+func (c *diskUsageDownloadCacheContext) Type() string {
+	c.AddHeader(typeHeader)
+	return "Download Cache"
+}
+func (c *diskUsageDownloadCacheContext) TotalCount() string {
+	c.AddHeader(totalHeader)
+	return fmt.Sprintf("%d", c.TotalNumber)
```

Comment:
This can be `return strconv.FormatInt(c.TotalNumber, 10)` (and similar for Active below). Should be functionally equivalent, but I find it nice to avoid `Sprintf` and the potential for format string problems that aren't caught at compile time.

Feel free to ignore.